source/screens/App/components/AppEntry.js

```diff
@@ -1,30 +1,33 @@
 import React from 'react'
-import {
+import { 
   BrowserRouter as Router,
   Switch,
-  Route,
+  Route
 } from 'react-router-dom'
+
 import Logo from './Logo'
+import { SideMenuBar } from '../components'
+import { Home } from '../screens/Home'
 import { Jobs } from '../screens/Jobs'
 import { Leads } from '../screens/Leads'
 import { Materials } from '../screens/Materials'
 import { Quotes } from '../screens/Quotes'
-import { UserProfile } from '../screens/UserProfile'
 import { Vendors } from '../screens/Vendors'
 import './AppEntry.css'
 
 const AppEntry = () => (
   <div className="AppEntry">
     <div className="AppEntry-Menu">
-      <Logo className="AppEntry-Logo" />
+      <Logo className="AppEntry-Logo"/>
+      <SideMenuBar />
     </div>
     <div className="AppEntry-Body">
       <Router>
         <Switch>
+          <Route exact path="/" component={Home} />
```

Comment:
The `/` should only take you to the login page

Reply:
Well, I just got down to where you are setting `/login` to take you to the log in page. I guess this one should do a check to see if you are logged in and redirect to either the login page or the UserProfile page, whichever is appropriate.

Reply:
Ok, my PR is working on the login page routing - these were questions I had before lol